memory.sh

```diff
@@ -10,6 +10,18 @@ peek() {
   eval "echo \$cell_$1"
 }
 
+lowercase() {
+  if test 65 -le "$1" -a $1 -le 90; then # 'A' <= $1 <= 'Z'
+    echo `"$expr" "$1" + 32` # $1 + 'a' - 'A'
+  else
+    echo "$1"
+  fi
+}
+
+peek_lowercase() {
```

Comment:
This convenience function seems oddly specific.  I'd prefer to just have indentifier() call lowercase() on a temporary variable.